setup.cfg

```diff
@@ -7,7 +7,7 @@ all_files = 1
 upload-dir = docs/_build/html
 show-response = 1
 
-[pytest]
+[tool:pytest]
```

Comment:
you need to change the minversion for pytest, too to 3+ otherwise this section won't be recognized for older pytests.